cli/src/pcluster/templates/queues_stack.py

```diff
@@ -335,6 +335,10 @@ def _add_compute_resource_launch_template(
                         self._shared_storage_attributes[SharedStorageType.EFS]["IamAuthorizations"],
                         use_lower_case=True,
                     ),
+                    "efs_accesspoint_ids": to_comma_separated_string(
```

Comment:
This change should be captured in unit test here: https://github.com/aws/aws-parallelcluster/blob/develop/cli/tests/pcluster/templates/test_queues_stack.py#L114